src/behavior.rs

```diff
@@ -147,7 +154,9 @@ pub trait Behavior<Pane> {
 
                 // Allocate
                 let close_btn_id = ui.auto_id_with("tab_close_btn");
-                let close_btn_response = ui.interact(close_btn_rect, close_btn_id, Sense::click());
+                let close_btn_response = ui
+                    .interact(close_btn_rect, close_btn_id, Sense::click_and_drag())
+                    .on_hover_cursor(self.close_button_hover_cursor());
```

Comment:
note that due to button grabbing the `drag` sense, dragging the close button does nothing (which I think is a good thing), so the user changing the cursor for it to anything but the default would be quite confusing imho

Reply:
My reasoning about this was: 

1. On the web something clickable usually uses a hand pointing cursor.
2. On desktop applications something clickable does not usually affect the cursor (rather leverages other visual feedback mechanisms)

I though that if someone were to make an application with egui & egui_tiles and intends to only publish it as a web application they might welcome having the option to customize it.

What do you think?

Reply:
An egui user can already change the default cursor for _all_ buttons using https://docs.rs/egui/latest/egui/style/struct.Visuals.html#structfield.interact_cursor

Reply:
I had completely forgotten about that! Thank you. Removing it.